src/style/mixins.scss

```diff
@@ -130,6 +130,19 @@
   padding: 0;
 }
 
+/// Visually hides an element but still allows it to be tabbed to.
+@mixin iui-visually-hidden {
```

Comment:
Shouldn't we have undo mixin? Like in https://github.com/iTwin/synchronization-report-react/blob/main/src/components/utils/VisuallyHidden.scss

Reply:
I've added the revert mixin.